pkg/cap/cap_test.go

```diff
@@ -0,0 +1,145 @@
+/*
+   Copyright The containerd Authors.
```

Comment:
If we don't move it to runtime-spec (see my other comment), we should at least accredit the original authors, and add a copyright comment `Copyright 2013 Suryandaru Triandana <syndtr@gmail.com>` from https://github.com/syndtr/gocapability/blob/master/LICENSE

Reply:
`pkg/cap` is not a copy of ` syndtr/gocapability`

Reply:
🤦 i thought you were moving that code in. My bad, shouldn't read this from my phone